src/Misc/layoutbin/de-DE/strings.json

```diff
@@ -394,6 +396,7 @@
   "ListenForJobs": "{0:u}: Auf Aufträge lauschen",
   "LocalClockSkewed": "Die Uhr des lokalen Computers weicht möglicherweise um mehr als fünf Minuten von der Serverzeit ab. Synchronisieren Sie Ihre Uhr mit Ihrer Domäne oder Internetzeit, und versuchen Sie es noch einmal.",
   "LocalSystemAccountNotFound": "Das lokale Systemkonto wurde nicht gefunden.",
+  "LogOutputMessage": "Der Agent hat das Hochladen von Protokollen sowie das Speichern von Protokollen in einer Datei aktiviert. Nachdem der Job abgeschlossen ist, können Sie die Protokolle dieses Schritts unter {0} auf dem Agent abrufen.",
```

Comment:
Please do not include localization in your PR. We localization flow that will automatically provide valid localization

Reply:
Ok do you want me to remove it? Also, is there any further action I need to take?

Reply:
Removed what was added in the strings files